.travis.yml

```diff
@@ -49,15 +47,15 @@ deploy:
     secret_access_key: $AWS_SECRET_ACCESS_KEY
     bucket: $ARTIFACTS_BUCKET
```

Comment:
change to $AWS_S3_BUCKET

This variable name is left over from when I was trying to upload the docs to s3 as build artifacts.